File: Beecrowd_Parte1/AUnicaChance_2496.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n); // Número de casos de teste
    
    for (int i = 0; i < n; i++) {
        int m;
        scanf("%d", &m); // Tamanho da string s
        
        char s[m+1]; // String s (mais 1 para o '\0')
        scanf("%s", s); // Leitura da string s
        
        int trocas = 0;
        for (int j = 0; j < m; j++) {
            char expectedChar = 'A' + j;
            if (s[j] != expectedChar) {
                trocas++;
            }
        }
        
        if (trocas < 3) {
            printf("There are the chance.\n");
        } else {
            printf("There aren't the chance.\n");
        }
    }
    
    return 0;
}

File: Beecrowd_Parte1/AUnicaChance_2496.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int n = scanner.nextInt(); // Número de casos de teste
        
        for (int i = 0; i < n; i++) {
            int m = scanner.nextInt(); // Tamanho da string s
            String s = scanner.next(); // String s
            
            int trocas = 0;
            for (int j = 0; j < m; j++) {
                char expectedChar = (char) ('A' + j);
                if (s.charAt(j) != expectedChar) {
                    trocas++;
                }
            }
            
            if (trocas < 3) {
                System.out.println("There are the chance.");
            } else {
                System.out.println("There aren't the chance.");
            }
        }
        
        scanner.close();
    }
}
